source/command-line-options.txt

```diff
@@ -33,18 +35,77 @@ Use the following syntax to start |compass| and set a configuration option:
 Options
 -------
 
-You can set the following configuration options using the command line:
+You can set these options when using the command line:
```

Comment:
Are there links available for these topics (like on line 55)?

Reply:
I'll take a closer look. Not sure about many of these.

Reply:
I've linked out where I could find additional info on the Compass docs.